### DISCUSSION TITLE
ADE 4.X workaround for mac?

### DISCUSSION THREAD

**Discussion by sndrake** [apprenticeharper/DeDRM_tools#2023] [Q&A]
Apologies if there's a thread I'm not reading to answer this question; TIA in directing me to it!

ADE 2.0.1 is no longer available for the 64-bit mac OSes, but DeDRM has worked sometimes with ADE 4.X (currently have ADE 4.5 installed). Other times, like today, I download the .acsm, open in ADE, and then add the PDF to calibre. Everything seems to work as usual except the PDF comes out with the correct number of blank pages. Log is attached below. Other than trying to find something to run ADE 2.0.1, is there a solution?

[tmpdlpr4_tg.txt](https://github.com/apprenticeharper/DeDRM_tools/files/8144986/tmpdlpr4_tg.txt)

**Comment by ElleKayEm** [comment]
This may be the hardened Adobe DRM which cannot be removed.  Suggest trying noDRM's DeDRM which, if nothing else, will give a more informative error message.  Available here:  https://github.com/noDRM/DeDRM_tools

**Reply by ZolaLa9** [reply]
You can also try Leseratte10's ACSM Input plugin to emulate ADE 2.0.1 on Mac: https://github.com/Leseratte10/acsm-calibre-plugin
